Automattic/wp-calypso
# Gutenboarding: Consolidate templates metadata

This PR removes the "available-designs.json" file on gutenboarding to rely on the "page-templates.json", higher-level data. Those two files where defining the same content, so there isn't any reason to keep both of them. 

It also removes (temporarily) the vertical param on the mshots call on the designs page. Since the call for testing is not going to include vertical content, at this point the only thing it's doing is being a cache buster for shots, so we can remove it by now until it actually has some use. 

How to test?
==========
0. Go to http://calypso.localhost:3000/gutenboarding , enter your site data and get to the design step.
1. You should be seeing 10 options there and be able to choose one.
